kolibri_explore_plugin/assets/src/views/ChannelsPage.vue

```diff
@@ -1,41 +1,29 @@
 <template>
 
-  <div class="channels">
-    <PageHeader
-      :title="coreString('channelsLabel')"
-      class="visuallyhidden"
-    />
-
-    <div class="searchbar">
-      <label class="visuallyhidden" for="searchfield">{{ coreString('searchLabel') }}</label>
-      <input
-        id="searchfield"
-        ref="searchInput"
-        v-model.trim="searchQuery"
-        type="search"
-        class="search-input"
-        dir="auto"
-        :placeholder="coreString('searchLabel')"
-        @focus="searchFocus = true"
-        @blur="searchFocus = false"
-      >
-
-      <KIconButton
-        icon="search"
-        :appearance="searchAppareance()"
-        @click="filter"
+  <b-container class="channels mb-3 mt-3">
+    <b-card-group
```

Comment:
b-card-group are not responsive. I guess you are trying to do the same as in the template for CardRow, so it's fine for now.